pyccel/parser/semantic.py

```diff
@@ -902,7 +902,7 @@ def _handle_function_args(self, arguments, **settings):
                 args.append(a)
         return args
 
-    def _handle_function(self, func, args, **settings):
+    def _handle_function(self, expr, func, args, **settings):
         if not isinstance(func, (FunctionDef, Interface)):
             args, kwargs = split_positional_keyword_arguments(*args)
             for a in args:
```

Comment:
Could you please add a docstring to this function? It has become really obscure.

Reply:
Fixed in 008e068